packages/form-js-editor/src/render/components/properties-panel/entries/KeyEntry.js

```diff
@@ -1,19 +1,49 @@
+import { isUndefined } from 'min-dash';
+
+import useService from '../../../hooks/useService';
+
 import { TextInputEntry } from '../components';
 
-// TODO: Check whether key unique
 export default function KeyEntry(props) {
   const {
     editField,
     field
   } = props;
 
+  const formFieldRegistry = useService('formFieldRegistry');
+
+  const validate = (value) => {
+    if (isUndefined(value) || !value.length) {
+      return 'Key must not be empty.';
+    }
+
+    if (/\s/.test(value)) {
```

Comment:
As the context is given we could consider to drop _Key_ as a work (i.e. simplify to _Must not be empty_.

Reply:
I've used the same wording as in the new properties panel (cf. https://github.com/bpmn-io/properties-panel/pull/75). What should we do?

Reply:
Well, [my feedback](https://github.com/bpmn-io/form-js/pull/127#discussion_r667529449) applies for that one, too. (CC @pinussilvestrus, CC @andreasgeier).

Good to have it consistent to start with I guess.